docs/docs-content/clusters/cluster-management/cluster-updates.md

```diff
@@ -140,3 +142,123 @@ You can follow these steps to validate all cluster update approaches.
 3. Select the cluster you updated, and navigate to the **Profile** tab of the cluster.
 
 4. Confirm that the cluster is now using an updated profile.
+
+## Palette Reconciliation Behavior
+
+When cluster profile updates are applied, Palette will apply the changes to the cluster but will not delete any
+resources that are no longer in use. Depending on the situation, you can choose to keep or remove these resources. To
+remove the resources, utilize one of the following two methods.
+
+The first method would be a clean deployment with a new cluster profile. Once the new cluster and cluster profile are
+deployed, the previous version could be deleted. The challenge, however, is when particular resources are in use.
+
+The second method is to use `kubectl` to scale the resource count to 0. While deleting the resource using something like
+K9s is possible, Palette will recreate it immediately. Scaling replicas to zero via `kubectl` ensures the resource is
+effectively removed.
+
+This section provides guidance on removing resources using the second method.
+
+This section provides guidance on removing the resources using the second method.
+
+### Prerequisites
+
+- An active Kubernetes cluster in Palette.
+- A terminal client.
+
+### Enablement
+
+The following steps provide an example of how to clean up unneeded resources using `kubectl`. In this example, an AWS
+cluster called **cluster-update-deletion** is created using a cluster profile with the
+[Hello Universe](/integrations/packs/?pack=hello-universe) included. The Hello Universe pack has the API preset option
```

Comment:
```suggestion
<VersionedLink text="Hello Universe" url="/integrations/packs/?pack=hello-universe"/> included. The Hello Universe pack has the API preset option
```

Reply:
Any links to packs need to be VersionedLinks as per this suggestion.

https://github.com/spectrocloud/librarium?tab=readme-ov-file#link-to-a-pack